roles/platform/tasks/setup_gcp_env.yml

```diff
@@ -34,3 +34,4 @@
     tags: "{{ plat__tags }}"
     freeipa:
       instanceCountByGroup: "{{ plat__env_freeipa }}"
+    zones: "{{ plat__gcp_availability_zones | length > 0 | ternary(plat__gcp_availability_zones, omit) }}"
```

Comment:
`()`'s are required around the `plat__gcp_availability_zones | length > 0` expression.
Without I get this error:
```
"Unexpected templating type error occurred on ({{ plat__gcp_availability_zones | length > 0 | ternary(plat__gcp_availability_zones, omit) }}): '>' not supported between instances of 'int' and 'str'"}
```